scale/docs/rest/v6/scale_file.rst

```diff
@@ -0,0 +1,418 @@
+
+.. _rest_v6_scale_file:
+
+v6 Scale File Services
+========================================================================================================================
+
+These services provide access to information about general files that are being tracked by Scale.
+
+.. _rest_v6_scale_file_list:
+
+v6 Scale Files
+----------------------
+
+**Example GET /v6/files/ API call**
+
+Request: GET http://.../v6/files/
+
+Response: 200 OK
+
+ .. code-block:: javascript  
+    { 
+        "count": 55, 
+        "next": null, 
+        "previous": null, 
+        "results": [ 
+            { 
+                "id": 465, 
+                "workspace": { 
+                    "id": 2, 
+                    "name": "Products" 
+                }, 
+                "file_name": "my_file.kml", 
+                "media_type": "application/vnd.google-earth.kml+xml", 
+                "file_size": 100, 
+                "data_type": [], 
+                "is_deleted": false, 
+                "url": "http://host.com/file/path/my_file.kml", 
+                "created": "1970-01-01T00:00:00Z", 
+                "deleted": null, 
+                "data_started": null, 
+                "data_ended": null, 
+                "geometry": null, 
+                "center_point": null, 
+                "countries": ["TCY", "TCT"], 
+                "last_modified": "1970-01-01T00:00:00Z", 
+                "file_path": "path/to/the/file.png",
+                "is_operational": true, 
+                "is_published": true, 
+                "has_been_published": true, 
+                "published": "1970-01-01T00:00:00Z", 
+                "unpublished": null, 
+                "source_started": "1970-01-01T00:00:00Z", 
+                "source_ended": "1970-01-02T00:00:00Z", 
+                "job": { 
+                    "id": 47 
+                }, 
+                "job_exe": { 
+                    "id": 49 
+                },
+                "job_output": "output_name_1",
+                "job_type": { 
```

Comment:
Switch job type to its base serializer (go ahead and create a v6 base job type serializer).